### PR TITLE
fix(webdriverio): work with @wdio/globals

### PR DESCRIPTION
**Pull request by straker**
When using `@wdio/globals`, `browser` is a function and not an object (thanks wdio).

Closes: https://github.com/dequelabs/axe-core-npm/issues/798

### CODE REVIEW

File: test/wdio/wdio-globals.spec.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+describe('@wdio/globals', () => {
+  let server: Server;
+  let addr: string;
+  let builder: AxeBuilder;
+  before(async () => {
+    const app = express();
+    app.use(express.static(fixturesPath));
+    server = createServer(app);
+    addr = await testListen(server);
```
Comment (Zidious) [inline test/wdio/wdio-globals.spec.ts:17]: I know we use `testListen` in the other packages for their tests, it appears it's now deprecated and moved over to [`aync-listen`](https://github.com/vercel/async-listen). We should update it here and create a tech-debt to update it in all the other packages?

File: test/wdio/wdio-globals.spec.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  });
+
+  beforeEach(() => {
+    // node v20 causes a typescript error due to our current type of WebdriverIO.Browser not being compatible with Browser from webriverio
+    builder = new AxeBuilder({ client: browser as any });
+  });
```
Comment (Zidious) [inline test/wdio/wdio-globals.spec.ts:23]: NBD - reference https://github.com/dequelabs/axe-core-npm/issues/883 so we know to update it when this gets picked up

File: packages/webdriverio/src/utils.ts [hunk 3]
```diff
@@ -16,7 +16,7 @@ export const FRAME_LOAD_TIMEOUT = 1000;
  * Validates that the client provided is WebdriverIO v5 or v6.
```
Comment (Zidious) [inline packages/webdriverio/src/utils.ts:16]: This comment is outdated we should say something like >= 5

**Review by Zidious — CHANGES_REQUESTED** [review]
One minor comment about test-listen otherwise LGTM

File: test/wdio/wdio-globals.spec.ts (hunk 1 above)
Comment (straker) [reply test/wdio/wdio-globals.spec.ts:17]: I think that work should be done completely in a different pr.

Comment (straker) [reply test/wdio/wdio-globals.spec.ts:17]: https://github.com/dequelabs/axe-core-npm/issues/884

**Review by Zidious — APPROVED** [review]
LGTM

**Pull request merged**